python/opae.admin/opae/admin/tools/rsu.py

```diff
@@ -70,27 +62,75 @@
      of 25:00.0.
      NOTE: Both BMC and FPGA images will be reconfigured from user bank.
 
-     %(prog)s bmcimg 25:00.0 -f
```

Comment:
Can we maybe put an example usage with nextboot?

Reply:
fixed